cmd/aries-js-worker/src/aries.js

```diff
@@ -53,6 +53,24 @@ async function invoke(pkg, fn, arg, msgTimeout) {
     })
 }
 
+async function waitForNotification(topics) {
+    return new Promise((resolve, reject) => {
+        const timer = setTimeout(_ => resolve(), 10000)
```

Comment:
We should make 10000 a const and supply an explanation on what it is and why 10000.

Reply:
I will add explanation